[2022-06-27 13:32:56,443] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dag1.model_C scheduled__2022-05-22T00:00:00+00:00 [queued]>
[2022-06-27 13:32:56,454] {taskinstance.py:1159} INFO - Dependencies all met for <TaskInstance: dag1.model_C scheduled__2022-05-22T00:00:00+00:00 [queued]>
[2022-06-27 13:32:56,456] {taskinstance.py:1356} INFO - 
--------------------------------------------------------------------------------
[2022-06-27 13:32:56,457] {taskinstance.py:1357} INFO - Starting attempt 1 of 1
[2022-06-27 13:32:56,459] {taskinstance.py:1358} INFO - 
--------------------------------------------------------------------------------
[2022-06-27 13:32:56,490] {taskinstance.py:1377} INFO - Executing <Task(PythonOperator): model_C> on 2022-05-22 00:00:00+00:00
[2022-06-27 13:32:56,496] {standard_task_runner.py:52} INFO - Started process 58226 to run task
[2022-06-27 13:32:56,503] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'dag1', 'model_C', 'scheduled__2022-05-22T00:00:00+00:00', '--job-id', '712', '--raw', '--subdir', 'DAGS_FOLDER/dag1.py', '--cfg-path', '/tmp/tmps9ut45r4', '--error-file', '/tmp/tmpc2h7qovp']
[2022-06-27 13:32:56,505] {standard_task_runner.py:80} INFO - Job 712: Subtask model_C
[2022-06-27 13:32:56,725] {task_command.py:370} INFO - Running <TaskInstance: dag1.model_C scheduled__2022-05-22T00:00:00+00:00 [running]> on host 43f8d36cfb0f
[2022-06-27 13:32:57,233] {taskinstance.py:1571} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=dag1
AIRFLOW_CTX_TASK_ID=model_C
AIRFLOW_CTX_EXECUTION_DATE=2022-05-22T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-05-22T00:00:00+00:00
[2022-06-27 13:32:57,239] {python.py:173} INFO - Done. Returned value was: 7
[2022-06-27 13:32:57,416] {taskinstance.py:1400} INFO - Marking task as SUCCESS. dag_id=dag1, task_id=model_C, execution_date=20220522T000000, start_date=20220627T133256, end_date=20220627T133257
[2022-06-27 13:32:57,502] {local_task_job.py:156} INFO - Task exited with return code 0
[2022-06-27 13:32:57,687] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
